puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-infra-update-jenkins-jobs.yml

```diff
@@ -0,0 +1,10 @@
+---
+- job:
+    name: 'foreman-infra-update-jenkins-jobs'
```

Comment:
I think this name would need to be something more like `foreman-infra-jenkins-jobs-update` to follow our naming scheme